sources/packages/web/src/views/aest/institution/Profile.vue

```diff
@@ -79,15 +78,19 @@
           </content-group></v-col
         >
       </v-row>
-      <p class="category-header-large color-blue mt-2 mb-2">Mailing address</p>
+      <p class="category-header-medium mt-5">Mailing address</p>
       <content-group>
         <title-value
           propertyTitle="Address 1"
           :propertyValue="institutionProfileDetail.mailingAddress?.addressLine1"
         />
         <title-value
           propertyTitle="Address 2"
-          :propertyValue="institutionProfileDetail.mailingAddress?.addressLine2"
+          :propertyValue="
+            emptyStringFiller(
```

Comment:
IMO this composable can be generic to replace the empty string with what we want. e.g 
```
emptyStringFiller(addressline2,"-");
emptyStringFiller(addressline2,"Not Provided");
```

Reply:
👍 i do agree, but for now as per the fillers we have only '-' also we have a constant defined in the formatters, so using it as it is, like the other y/n composable. In the future we can add parameters to the method, if needed. Thanks for the idea.